prem/HolyGrail.html

```diff
@@ -0,0 +1,82 @@
+<!-- <!DOCTYPE html> -->
+<html>
+<head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>
+        HolyGrail
+    </title>
+    <style>
+        body {
+            overflow: hidden;
+        }
+        .head,.foot,.sidebar1,.sidebar2,.article{
+            background-color: #D3D3D3;
+            font-weight:bold;
+        }
+
+        .head,.foot{
+            height: 19%;
+            text-align: center;
+            box-sizing: border-box;
+            padding: 4%;
+        }
+        .head {
+            margin-bottom:0.5%;
+        }
+        .foot {
+            margin-top:0.5%;
+            margin-bottom: 0;
+        }
+        .content {
+            height: 60%;
+        }
+        .sidebar1,.sidebar2{
+            width: 22%;
+            height: 100%;
+            box-sizing: border-box;
+            text-align: center;
+            padding-top:12.5%;
+        }
+        .sidebar1{
+            float: left;   
+        }
+        .sidebar2{
+            float: right;
+        }
+        .article{
+            position:absolute;
+            top:0;

```

Comment:
What is the need for having top,bottom,right, left as 0?

Reply:
Basically in order to make a element center of the page , we need to divide the all four positions (top,left,right,bottom). If its done manually, we cant able to make it responsive to the different screens. So all the four positions are initially set to 0, so that it is located on the top-left corner of the document(which is basically the origin point). From the origin point it is easier to position the element to the center